lib/hako/container.rb

```diff
@@ -87,6 +87,32 @@ def log_configuration
       end
     end
 
+    # @return[Hash, nil]
+    def health_check
+      if @definition.key?('health_check')
+        conf = @definition['health_check']
+        ret = {
+          command: conf.fetch('command'),
+        }
+
+        # optional
+        if conf.key?('interval')
+          ret[:interval] = conf.fetch('interval')
```

Comment:
The default value must be specified because DescribeTaskDefinition returns interval=30 even if RegisterTaskDefinition is called without interval option. timeout and retries also have their default value.
https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html
Otherwise EcsDefinitionComparator claims that the task definition and Hako definition are always different.